### ISSUE TITLE
Container assumes using eth0 as network interface

### ISSUE THREAD

**Issue by Lastin** [haugene/docker-transmission-openvpn#1885]
## Before creating this issue I have:

**REQUIRED**
- [x] Read through the pinned issues for related problems
- [x] Searched for similar [provider issues][pi] and [container issues][ci]
- [x] [Read the documentation][rd], especially the troubleshooting section and FAQ
- [x] Tried to add as much relevant information to the issue as possible
- [x] Verified I have tried using newest release as well
- [x] Agreed that my issue will be closed if I do not follow this template and will remain closed until I complete the template

[pi]: https://github.com/haugene/vpn-configs-contrib/issues
[ci]: https://github.com/haugene/docker-transmission-openvpn/issues
[rd]: https://haugene.github.io/docker-transmission-openvpn/

### Container version & last working release

**Required, problem occurs in :** 
haugene/transmission-openvpn@sha256:228ba95784bf4a2a36c51fd82559cbda677a98b033f1575b28feabda4ff749cc

*If possible, last working version:* 
 
latest

### Describe the problem

**REQUIRED**
```txt
No internet connection when connected through eth1 interface.
Plugging the cable to interface eth0 connection works and VPN starts as expected.
```
### Describe the steps you have tried to solve the problem

**REQUIRED**
```txt
1) Read entire documentation
2) Spotted mentioned 172.x.0.0/16 dev eth0 in "debugging" section
3) Figured it assumes eth0
4) Plugged the cable from eth1 into eth0
```

### Add your docker run command or docker-compose file or env details

**REQUIRED**
```txt
services:
    t-vpn:
        cap_add:
            - NET_ADMIN
        volumes:
            - '/mnt/asdf123/data:/data'
            - '/home/user/vpnstuff/custom:/etc/openvpn/custom/'
        environment:
            - OPENVPN_PROVIDER=custom
            - OPENVPN_CONFIG=openvpn
            - OPENVPN_USERNAME=foo
            - OPENVPN_PASSWORD=bar
            - LOCAL_NETWORK=192.168.1.0/24
        logging:
            driver: json-file
            options:
                max-size: 10m
        ports:
            - '9091:9091'
        image: haugene/transmission-openvpn
```

### Logs

 

**REQUIRED**
```txt
Lost them, sorry.
Basically ip of VPN provider: connection refused
```

### Host system

**REQUIRED**
```txt
Ubuntu 20.04
```

**Comment by pkishino** [comment]
I think this might be more docker related/based on the network stack.. as you manually can assign any network to the container it will default to the default one provided by docker

**Comment by pkishino** [comment]
I believe this can be closed
